docs/community/working-group.md

```diff
@@ -0,0 +1,189 @@
+With the introduction of Draft PRs on GitHub, we are reorienting the acceptance process around the PR itself.
+Going forward, an RFC will have three stages:
+
+1. A Draft PR, denoting that the RFC is still in the review period, openly soliciting comments,
+   and that it may continue to be significantly iterated upon with revisions, edits, and responses to
+   community feedback or concerns.
+1. After two months of discussion, the RFC/PR author marks the PR as
+   [ready for review](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-stage-of-a-pull-request),
+   upon which the Committee will review the entirety of the RFC and RFC PR comments,
+   discuss the contents, and make a decision on whether it is reasonable to pursue.
+   If after this review, the Committee determines that the intent of the RFC is not reasonable
+   (e.g. there may be unreconcilable issues with the design,
+   or the intent may not fit with the principles of PowerShell),
+   they will reject the PR and the process terminates.
+   should be merged as an accepted RFC.
+1. In most cases, the Committee will choose to wait for the code PR to be merged as experimental,
+   and leverage user feedback or telemetry to determine if the RFC PR should be merged.
+1. Finally, the Committee will choose to either merge or close the RFC PR,
+   marking the RFC as either accepted or rejected, respectively.
+
+#### Prototypes
+
+Often times, implementing an idea can demonstrate opportunities or challenges that were not well
+understood when the idea was formulated.
+Similarly, a "simple" code change can have far reaching effects that are not well understood
+until you're able to experiment with an idea within working code.
+
+To that end, it's required that *some* implementation exist before the Committee will consider an
+RFC for acceptance.
+That way, contributors can compile the PR branch to play with a working iteration of the idea
+as a way to understand whether the feature is working as expected and valuable.
+
+In most cases, as long as an RFC has already been written and published as a draft,
+Working Groups or the Committee will approve the feature for incorporation as an experimental feature,
+so that it can be trialed with greater usage as part of a preview release.
+In addition to increasing the scope of those who can provide real-world feedback on the feature,
+this enables us to use telemetry to understand if users are turning off the feature in large numbers.
+
+Note: today, this will be done on a case-by-case basis, but over time, the Committee will establish
+firmer guidelines around when PRs should be merged as experimental.
+
+Prototypes should be complete to the extent that they serve as reasonable indicators of the user experience.
+In the case that breaking changes are required of the feature, the break should be made in the prototype
+so that users can experiment with whether or not the break has a significant negative effect.
```

Comment:
@joeyaiello we should probably explicitly ask that this set of processes be re-reviewed, as being last in a long PR it seems to have been given the least attention (I say this guilty of it myself).

Reply:
Fair. To be honest, it probably received the least attention from my writing for the same reason.

I'll have some folks go over the verbiage tomorrow.